packages/runtime/container-runtime-definitions/src/containerRuntime.ts

```diff
@@ -75,14 +74,6 @@ export interface IContainerRuntime
 	 */
 	readonly attachState: AttachState;
 
```

Comment:
Do we have a grasp of who the intended consumer is for `IContainerRuntime`?  The doc comment on the interface gives no context.  Would be great to get clarity on that and write it down.

Reply:
The short answer is only the container author, the one who calls loadContainerRuntime gets the IContainerRuntime, so it is avaialbe to the entry point. All datastores just get IContainerRuntimeBase. However, it is very common to just cast to IContainerRuntime or ContainerRuntime to get whatever they need, so the distinction today is a bit arbitrary.